docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -57,6 +57,29 @@ You can use the YugabyteDB Anywhere UI to enable LDAP authentication for Yugabyt
 
   - The **Bind DN** field value represents the distinguished name (DN) used for searching and binding.
 
+### Map LDAP groups to YugabyteDB Anywhere roles
+
+You can map LDAP groups to YugabyteDB Anywhere roles. Upon login, the user's role is assigned based on whatever role their LDAP group has been mapped to in YugabyteDB Anywhere. If a user is in multiple mapped LDAP groups, YugabyteDB Anywhere performs a union of all the roles that are chosen as a result of group mapping and assigns the union to the user.
+
+To map LDAP groups to YugabyteDB Anywhere roles, do the following:
+
+1. Navigate to **Admin > User Management > User Authentication** and select **LDAP Configuration**.
+
+1. Under **Role Settings**, enable the **Define Role to Group Mapping** option.
```

Comment:
This will be "Map YugabyteDB Anywhere built-in roles to your existing LDAP groups" on master soon.